gcp/kbn-dev.sh

```diff
@@ -43,7 +43,7 @@ if [[ $TYPE && $TYPE == "pr" ]]; then
 elif [[  $TYPE && $TYPE == "tag" ]]; then
   BRANCH="tags/${VALUE} -b tags-${VALUE}"
   GCP_NAME="kbn-dev-v1-${TYPE}-${VALUE//./-}"
-elif [[  $TYPE && $TYPE == "BRANCH" ]]; then
```

Comment:
think I converted variables to uppercase during development, but this ain't no variable :)